src/displayapp/screens/Alarm.h

```diff
@@ -62,8 +66,16 @@ namespace Pinetime {
         void HideInfo();
         void ToggleRecurrence();
         void UpdateAlarmTime();
+        void UpdateStopProgress();
         Widgets::Counter hourCounter = Widgets::Counter(0, 23, jetbrains_mono_76);
         Widgets::Counter minuteCounter = Widgets::Counter(0, 59, jetbrains_mono_76);
+
+        lv_obj_t* progressStop;
+        bool stopBtnPressing = false;
+        TickType_t stopBtnPressTime = 0;
```

Comment:
This could be `std::optional<TickType_t>`, where `nullopt` means not pressing and any integer value is the current stop pressing time